Tasks/AutomatedAnalysisV0/task.json

```diff
@@ -0,0 +1,468 @@
+{
+    "id": "6B519857-1A20-4248-BAE8-AAD039015AFB",
+    "name": "AutomatedAnalysis",
+    "friendlyName": "Automated Analysis",
+    "description": "Do a automated analysis of your appication metrics ",
+    "author": "Microsoft Corporation",
+    "helpUrl": "https://docs.microsoft.com/azure/devops/pipelines/tasks/utility/azure-monitor",
+    "helpMarkDown": "[Learn more about this task](https://go.microsoft.com/fwlink/?linkid=870240)",
+    "category": "Utility",
+    "releaseNotes": "What's new in Version 1.0:<br/>&nbsp;Added support for query unified Azure monitor alerts.",
+    "visibility": [
+        "Build",
+        "Release"
+    ],
+    "runsOn": [
+        "Server"
+    ],
+    "version": {
+        "Major": 1,
```

Comment:
"Major": 0

Reply:
Why Major: 0? It's ready for general usage in my opinion. Any specific reason we wanna go in preview mode?

Reply:
I don't think 0 means preview. Don't we start with Major version 0 for all our tasks ?
Since we are at it, I think we should start in preview mode, as there are so many moving parts. We should wait to hear from customers and fix any issues before calling it stable.